pkg/ssh/shell/setupBuildEnv.sh

```diff
@@ -3,22 +3,3 @@ set -e
 #name:TASK [install gcc if not available]
 sudo apt-get -y -o DPkg::Lock::Timeout=120 update
```

Comment:
I wonder if you can do this concurrently? I don't think it'll be used by the installs?

Reply:
we can delete this actually as we do it in pkg/ssh/shell/setupNode.sh

Reply:
no we can't unfortunately